test/interface.jl

```diff
@@ -38,7 +38,10 @@ using Zygote: Grads
     x, y = [1,2], [1]
     ps = Params([x, y])
     @test length.(ps) == length.([x, y]) # 617
+    @test size.(ps, 1) == [2, 1]
     @test all(Params([[1,1]]) .== Params([[1,1]]))
+
+    @test_throws ArgumentError gradient(() -> sum(sum.(ps)), ps)
```

Comment:
Can drop this since we may want to enable this functionality with some changes in zygote.

Reply:
Maybe we want a test_broken here. In the end, this is a bug that we will want to fix at some point

Reply:
Someone make a suggestion. Write the test which would pass if this were fixed. But if someone just deletes the line making the error, then that should produce an error here.

Reply:
I think this is fine. It correctly tests the current behavior (most important), and when this is fixed, this will throw an error and be modified for the new behavior.